tests/test_config_entries.py

```diff
@@ -6586,53 +6590,3 @@ async def test_reauth_helper_alignment(
     # Ensure context and init data are aligned
     assert helper_flow_context == reauth_flow_context
     assert helper_flow_init_data == reauth_flow_init_data
-
-
-async def test_reconfigure_helper_alignment(
-    hass: HomeAssistant,
-    manager: config_entries.ConfigEntries,
-) -> None:
-    """Test `start_reconfigure_flow` helper alignment.
-
-    It should be aligned with `ConfigEntry._async_init_reconfigure`.
-    """
```

Comment:
This test was wrong - as it should NOT have been aligned with `ConfigEntry._async_init_reconfigure`